dace/codegen/targets/framecode.py

```diff
@@ -257,6 +258,13 @@ def generate_footer(self, sdfg: SDFG, global_stream: CodeIOStream, callsite_stre
         # Write closing brace of program
         callsite_stream.write('}', sdfg)
 
+        if sdfg.instrument is dtypes.InstrumentationType.GPU_TX_MARKERS:
```

Comment:
Please use `==` instead of `is` for future proofing. (applies to the rest of the cases in this file)